main.js

```diff
@@ -0,0 +1,131 @@
+let form = document.getElementById('add-form');
```

Comment:
-[ ] The project works, however, you didn't use classes to refactor your code.  Kindly do so.

That way, we will meet this particular requirement 💪